ietf-te-topology-packet.yang

```diff
@@ -83,11 +83,15 @@ module ietf-te-topology-packet {
      This version of this YANG module is part of RFC XXXX; see the
      RFC itself for full legal notices.";
 
-  revision 2018-06-22 {
```

Comment:
@xufengliu Is the version in the master branch is aligned with the version in the latest draft?